lib/ecto/adapters/postgres.ex

```diff
@@ -147,13 +147,20 @@ defmodule Ecto.Adapters.Postgres do
     end
   end
 
-  defp concat_if(content, nil, _fun),  do: content
-  defp concat_if(content, value, fun), do: content <> " " <> fun.(value)
+  defp concat_if(content, nil, _),  do: content
+  defp concat_if(content, false, _),  do: content
+  defp concat_if(content, value, fun) when is_function(fun), do: content <> " " <> fun.(value)
+  defp concat_if(content, _value, append), do: content <> " " <> append
```

Comment:
I don't think it is worth introducing a new case for this. You can make it be `fn _ -> "WITH (FORCE)" end`. :)

Reply:
Will fix.